### PR TITLE
awscli: 1.11.30 -> 1.11.35

### LINKED ISSUE #21591: awscli upgrade
Current version is 1.11.30 in nixpkgs. Newest version is 1.11.35.

### PR DESCRIPTION
**Pull request by spy-tech**
###### Motivation for this change
Fixes #21591 

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by spy-tech** [comment]
@FRidh TravisCI is not designed to handle this particular pull request. What are we going to do now?

**Comment by FRidh** [comment]
Just ignore Travis, its not reliable enough.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3074,35 +3071,35 @@ in {
...
 
   botocore = buildPythonPackage rec {
-    version = "1.4.87"; # This version is required by awscli
-    name = "botocore-${version}";
-
-    src = pkgs.fetchurl {
-      url = "mirror://pypi/b/botocore/${name}.tar.gz";
-      sha256 = "0fga1zjffsn2h50hbw7s4lcv6zwz5dcjgvjncl5y392mhivlrika";
-    };
+      version = "1.4.92"; # This version is required by awscli
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: We usually indent nix by two spaces, not four spaces. Please restore the original indentation.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -1543,12 +1541,11 @@ in {
...
-      description = "Unified tool to manage your AWS services";
-      license = stdenv.lib.licenses.asl20;
-      maintainers = with maintainers; [ muflax ];
-    };
-  };
+        homepage = https://aws.amazon.com/cli/;
+        description = "Unified tool to manage your AWS services";
+        license = stdenv.lib.licenses.asl20;
+        maintainers = with maintainers; [ muflax ];
+    };                                                                                                                                     };
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: the second parenthesis at the end of this line should got to the to the next line. You have seem to merge two lines by accident.

**Review by Mic92 — APPROVED** [review]
Thanks. Looks good to me except for the styling issues mentioned below.

**Comment by FRidh** [comment]
In the past (b2d9b48) I moved `awscli` to `python-packages.nix` because it is used as a library as well and therefore I removed the `namePrefix`. This was added back in 323eb0649929d1580ba1f8cc5812dba33f9e354b because when it is used as an application we typically do not include the prefix.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -1543,12 +1541,12 @@ in {
...
-      license = stdenv.lib.licenses.asl20;
-      maintainers = with maintainers; [ muflax ];
-    };
-  };
+        homepage = https://aws.amazon.com/cli/;
+        description = "Unified tool to manage your AWS services";
+        license = stdenv.lib.licenses.asl20;
+        maintainers = with maintainers; [ muflax ];
+    };           
+  };
```
Comment (grahamc) [inline pkgs/top-level/python-packages.nix]: Looks like a few more whitespace errors, then I'll merge :)

**Comment by spy-tech** [comment]
@grahamc I did what you asked for. Can you please merge it?

**Pull request closed without merging**